### PR TITLE
Update Satellite product versions for 6.10-beta

### PR DESCRIPTION
**Pull request by ryandeussing**
Changes in attributes, attributes-base, and docinfo files for each upstream title.

https://issues.redhat.com/browse/SATDOC-452

### CODE REVIEW

**Comment by ryandeussing** [comment]
Link checker is failing as the titles are not yet published. Seems like a chicken and egg problem but perhaps I have mis-assigned a version attribute. Thanks in advance for guidance here.

File: guides/common/attributes-base.adoc [hunk 1]
```diff
@@ -17,11 +17,11 @@
 :RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
 // For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6.9-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-6.8-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.9-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.9-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-rpms
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6-beta-rpms
+:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-6.9-rpms
```
Comment (ekohl) [inline guides/common/attributes-base.adoc:21]: This feels like a lot of repetition. We already have 6.9 so does this work?
```suggestion
:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-{ProductVersion}-rpms
```

File: guides/common/attributes-base.adoc [hunk 2]
```diff
@@ -17,11 +17,11 @@
 :RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
 // For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6.9-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-6.8-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.9-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.9-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-rpms
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6-beta-rpms
```
Comment (ekohl) [inline guides/common/attributes-base.adoc:20]: Can't we introduce a macro for the `6-beta` part that we later flip to `{ProductVersion}`?

Comment (ryandeussing) [reply guides/common/attributes-base.adoc:20]: @ekohl I'm not sufficiently comfortable doing this myself - can I create an issue for this enhancement for GA and prioritize merging this as-is? thanks

Comment (lzap) [reply guides/common/attributes-base.adoc:20]: It's challenging release for docs team, let's just do it the regular way, shall we?

File: guides/common/attributes-base.adoc (hunk 1 above)
Comment (lzap) [reply guides/common/attributes-base.adoc:21]: Please let's not tackle it in this PR to unblock @ryandeussing. Feel free to file a PR that can improve this, there is definitely room for improvement and I agree.

**Comment by lzap** [comment]
I will squash on merge.

**Pull request merged**